corehq/messaging/tasks.py

```diff
@@ -157,13 +158,10 @@ def _run_rule_sequentially():
             sync_case_for_messaging_rule.delay(domain, case_id, rule_id)
             incr += 1
             if incr >= 1000:
-                progress_helper.increase_total_case_count(incr)
```

Comment:
So we don't need `increase_total_case_count` anymore? If yes, we should remove it?

Reply:
One thing that `increase_total_case_count` was to update the expiry of the `total_key`, you would need to handle that else where otherwise it would expire and clear out during the run.

Reply:
Ohh yes. That was an oversight. I will correct it. 
Thanks for pointing it out 😄

Reply:
Thanks @mkangia. That is a good catch.
I have not thought about that. 
I have updated the `increase_total_case_count` with `update_total_key_expiry`.

Reply:
we need that when run for a shard as well, yes?

Reply:
Yes. Yes.
Done.